WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -525,9 +499,105 @@ private struct ProductsSection: View {
                      ]
                  )
             })
+        }
+    }
 
-            Divider()
+    var scanProductButton: some View {
```

Comment:
Nice! 💯 

We can also move it to a `private extension ProductsSection`

Reply:
Yeah, done in 7f743e179606eb5aae68ccdf0dd9080d353fc8a5